src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -71,36 +159,78 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
                 return;
             }
 
+            // NOTE: Values in TagHelperOutput.Attributes are already HtmlEncoded
+
             var content = new StringBuilder();
+            
+            if (modeResult.Mode == Mode.Fallback && string.IsNullOrEmpty(HrefInclude))
```

Comment:
`HreInclude` is for globbing the normal `href`. `FallbackHrefInclude` globs the fallback just fine.